plugins/atlas/apikey.go

```diff
@@ -0,0 +1,52 @@
+package atlas
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-env-variables/"),
```

Comment:
And would it make sense to use [this page](https://www.mongodb.com/docs/atlas/configure-api-access/) as the API Key's `DocsURL`?

```suggestion
		DocsURL:       sdk.URL("https://www.mongodb.com/docs/atlas/configure-api-access/"),
```